src/main/java/tech/jhipster/lite/module/domain/landscape/JHipsterLandscapeModule.java

```diff
@@ -84,6 +85,16 @@ public boolean equals(Object obj) {
     return new EqualsBuilder().append(module, other.module).isEquals();
   }
 
+  @Override
+  public String toString() {
+    return new ToStringBuilder(this)
```

Comment:
`SHORT_PREFIX_STYLE` is nicer, and is what we're already using in code when `ToStringBuilder` is used: 
```suggestion
    return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
```

Reply:
Done